README.md

```diff
@@ -15,3 +15,36 @@ As a user, I would like to be able to render content via a default output type,
 
 ## What's next?
 An output type provides a common backbone for all of our pages, so now we'll explore how we can start adding some semantic HTML and customization. The first concept to tackle in this regard is layouts. Switch to `fusion-03` to view a completed layout.
+
+If you are continuing this training from the last user story, then keep working on your current branch USERNAME-fusion-training.
+
+If you are starting this training from the third user story (task Fusion-03), then follow the steps under "What's next?" on the README file in the (https://github.com/wapopartners/Fusion-Training-User-Stories/blob/fusion-01/README.md)[fusion-01] branch - but checkout from fusion-02 instead.
+
+## New Task: User Story 03
+As a user, I would like to add a layout to my page, in order to give my page some structure and semantic HTML.
+
+## Guide
+1. All layouts are added to the `components/layouts` folder. Change directory into this folder and create a new find called `basic.jsx`
+
+2. In `basic.jsx`, create a new React component called `BasicLayout` that retrieves the following content from `props.children`:
+```
+const [header, main, aside, footer] = props.children;
```

Comment:
Yes, I agree that some more guidance would be helpful here. It is unclear that they will have to build the layout using classes such as `row`, `col-lg-12` etc.

Reply:
good point, I'll add in that blurb. I'm not sure if we need to go into details about using bootstrap classes though - devs should already be familiar with this. But it might help to mention something like `bootstrap can be used to style these components`

Reply:
yeah, I agree, just pointing out that's what we're using would be enough. It does look like Robin added bootstrap later on https://github.com/wapopartners/Fusion-Training-User-Stories/blob/fusion-05-update/components/output-types/default.jsx#L20

Reply:
I actually add that in the output type user story - one of the first ones I believe? On site, I typically explain that we are adding in bootstrap and that's how you can visually see the styles on the "welcome to fusion" headline.

Also, I've noticed that a lot of devs don't pick up on the fact that this line is just ES6 array destructuring and I have had to make a point of explaining many times that this line is using array destructuring to name variables that are coming out of the array, and that they have nothing to do with how the section names will be displayed in the PageBuilder Admin. Honestly, even though I like the array destructuring because it's readable, it might make sense to move that line completely and just use `props.children[0]`, `props.children[1]`, etc in it's place.

Reply:
I agree, although if the goal of the code in this repo is supposed to be what arc recommends/best practices - we should continue to use array destructuring and we can put in a comment above it with a link to docs on what destructuring is

Reply:
Agreed!

![approval](https://user-images.githubusercontent.com/13948125/91206124-a3930d00-e6d4-11ea-822c-5a21b9cd6086.gif)